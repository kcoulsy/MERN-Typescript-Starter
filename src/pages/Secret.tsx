import React from 'react';

const Secret: React.FunctionComponent = () => {
    return (
        <div>
            <h3>Secret Page</h3>
        </div>
    );
};

export default Secret;
