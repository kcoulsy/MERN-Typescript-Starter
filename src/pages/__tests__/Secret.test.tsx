import React from 'react';
import { shallow } from 'enzyme';

import Secret from '../Secret';
import { shallowToJson } from 'enzyme-to-json';

describe('render Secret component', () => {
    it('should render Secret page component', () => {
        const wrapper = shallow(<Secret />);

        expect(shallowToJson(wrapper)).toMatchSnapshot();
    });
});
