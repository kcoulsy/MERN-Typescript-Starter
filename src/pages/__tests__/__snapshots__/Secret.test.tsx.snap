// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Secret component should render Secret page component 1`] = `
<div>
  <h3>
    Secret Page
  </h3>
</div>
`;
